src/components/movie/list/MovieCard.tsx

```diff
@@ -62,13 +73,36 @@ const MovieCard: FC<IMovieCardProps> = props => {
 
     const posterSrc = poster_path || movieMockImgUrl;
 
+    const menuItems = [
```

Comment:
Let's move these items to constants

Reply:
I need functions from props here, it is very inconvenient to move whole array to consts. So I only turned string ids to local constants